gcloud-java-compute/src/test/java/com/google/gcloud/compute/ComputeImplTest.java

```diff
@@ -1907,6 +1928,148 @@ public void testCreateRegionAddressWithOptions() {
   }
 
   @Test
+  public void testCreateSnapshot() {
+    EasyMock.expect(computeRpcMock.createSnapshot(DISK_ID.zone(), DISK_ID.disk(),
+        SNAPSHOT_ID.snapshot(), null, EMPTY_RPC_OPTIONS)).andReturn(globalOperation.toPb());
```

Comment:
Nit: I think createSnapshot returned a zone operation instead of a global operation.  It doesn't really matter for the purpose of this test, but I do think we should change it because in general we want people to be able to look at test code/emulate test code to understand what's going on.